Samples/PublicSamples/RecordingBot/src/RecordingBot.Services/Http/LoggingMessageHandler.cs

```diff
@@ -1,19 +1,3 @@
-// ***********************************************************************
-// Assembly         : RecordingBot.Services
-// Author           : JasonTheDeveloper
-// Created          : 09-07-2020
-//
-// Last Modified By : dannygar
-// Last Modified On : 08-17-2020
-// ***********************************************************************
-// <copyright file="LoggingMessageHandler.cs" company="Microsoft Corporation">
-//     Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-// </copyright>
-// <summary>Helper class to log HTTP requests and responses and to set the scenario id based on the Scenario-ID or the X-Microsoft-Skype-Chain-ID headers
```

Comment:
Not a Ghost comment

Reply:
True, but the same comment is already on the class below as a proper comment